core/logic/src/main/java/org/apache/syncope/core/logic/TaskLogic.java

```diff
@@ -188,9 +189,9 @@
                     type, resourceDAO.find(resource), notificationDAO.find(notification), anyTypeKind, entityKey);
 
             List<T> result = taskDAO.findAll(
-                    type, resourceDAO.find(resource), notificationDAO.find(notification), anyTypeKind, entityKey,
-                    page, size, orderByClauses).stream().
-                    <T>map(task -> binder.getTaskTO(task, taskUtilsFactory.getInstance(type), details)).
+                            type, resourceDAO.find(resource), notificationDAO.find(notification), anyTypeKind, entityKey,
```

Comment:
Please revert this formatting-only change

Reply:
This has been reverted with 97328cb2d7229fcecf889294ee51fda5edd443ce